tests/unit/debug_extensions.cpp

```diff
@@ -25,46 +24,36 @@ TEST_F(NegativeDebugExtensions, DebugMarkerName) {
         GTEST_SKIP() << "Skipping object naming test with MockICD.";
     }
 
-    VkBuffer buffer;
-    VkDeviceMemory memory_1, memory_2;
     std::string memory_name = "memory_name";
 
     VkBufferCreateInfo buffer_create_info = vku::InitStructHelper();
     buffer_create_info.usage = VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT;
     buffer_create_info.size = 1;
-
-    vk::CreateBuffer(device(), &buffer_create_info, nullptr, &buffer);
+    vkt::Buffer buffer;
+    buffer.init_no_mem(*m_device, buffer_create_info);
```

Comment:
nit, do not bother if this is the only change, but we could do this:
```suggestion
    vkt::Buffer buffer(*m_device, buffer_create_info, vkt::no_mem);
```

Reply:
I like this.......like a lot better... let me do a global PR :laughing: (bright side if I do that, it will update all the copyrights in the process!)